packages/styles/framework/src/utilities/mixins/_mixins.scss

```
@@ -155,20 +211,48 @@
 
-    input:disabled,
-    input:disabled:checked {
-      & ~ label.#{$type}-label {
-        @include disabledRules;
+    &[aria-disabled="true"] {
+      @include disabledRules;
+    }
+
+    &[aria-checked="true"] {
+      cursor: default;
+      border-color: getColor('main');
```

Stylelint (17.3.0)

[error] Expected "getColor" to be "getcolor" (function-name-case)

(function-name-case)